src/_data/site.json

```diff
@@ -8,8 +8,8 @@
         "d2fe": "https://boards.greenhouse.io/flowfuse/jobs/5185319004"
     },
     "messaging": {
-        "tagLine": "Elevate Node-RED with FlowFuse ",
-        "subtitle": "FlowFuse provides a complete platform for building, scaling and securing your production Node-RED applications. Accelerate digitalization of your industrial processes with ease.",
+        "tagLine": "The Fastest Way to Build and Operate Industrial Data Solutions",
+        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. We help manage, scale and secure your Node-RED solutions.",
```

Comment:
Not sure about "we", maybe FlowFuse instead?

Reply:
yes we can change it to FlowFuse

Reply:
```suggestion
        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. FlowFuse helps manage, scale and secure your Node-RED solutions.",
```